lua/class_definitions.lua

```diff
@@ -347,6 +347,9 @@ classes = {
             { name = "fling_creature", rval = nil, args = { "Creature", "int", "float" } },
             { name = "fling_creature", rval = nil, args = { "Creature", "int", "float", "bool" } },
             { name = "get_cur_om", rval = "overmap&", args = { } },
+            { name = "get_cur_om_id", rval = "string", args = { } },
```

Comment:
Why are these functions part of the `game` class? They don't need the `game` class, they can be free functions implemented in "catalua.cpp" (like `game_items_at` or `get_monster_types`).